Documentation/devel/architecture.md

```diff
@@ -53,19 +53,18 @@ At this point the stage0 execs `/stage1/rootfs/init` with the current working di
 
 ### Stage 1
 
-The next stage is a binary that the user trusts to set up cgroups, execute processes, and perform other operations as root on the host. This stage has the responsibility of taking the execution group filesystem that was created by stage 0 and creating the necessary cgroups, namespaces and mounts to launch the execution group:
+The next stage is a binary that the user trusts to set up cgroups, execute processes, and perform other operations as root on the host. This stage has the responsibility of taking the execution group filesystem that was created by stage 0 and creating the necessary cgroups, namespaces and mounts to launch the execution group. Specifically, it must:
 
-- Generate systemd unit files from the Image and Pod Manifests. The Image Manifest defines the default `exec` specifications of each application; the Pod Manifest defines the ordering of the units, as well as any `exec` overrides.
-- (containing, respectively, the exec specifications of each pod and the ordering given by the user)
-- Set up any external volumes (undefined at this point)
-- nspawn attach to the bridge and launch the execution group systemd
-- Launch the root systemd
-- Have the root systemd
+- Read the Image and Pod Manifests. The Image Manifest defines the default `exec` specifications of each application; the Pod Manifest defines the ordering of the units, as well as any `exec` overrides.
```

Comment:
Pod Manifest actually supports overriding ALL parameters of the `app` section of Image Manifests now, not just exec, so please update to reflect that.